### PR TITLE
Set commit var to Heroku CI commit env var

### PR DESCRIPTION
**Pull request by pablofullana**
This PR sets `commit` variable to Heroku CI `HEROKU_TEST_RUN_COMMIT_VERSION` environment variable. Without this variable being set, coverage report upload was failing and with the following error message:
```
{"error": {"context": null, "reason": "missing required properties: ['commit']"}, "meta": {"status": 400}}
```

### CODE REVIEW

**Comment by jackmarchant** [comment]
This looks like it works on Heroku CI (I downloaded the file and ran it manually in our Heroku CI). Is there a plan to get this merged, or are you looking for help to fix the failing tests?

**Comment by jackmarchant** [comment]
It looks like tests are failing on master anyway, so merging this change shouldn't really change anything?

**Comment by pablofullana** [comment]
hey guys, any updates on this one?

**Comment by pablofullana** [comment]
@stevepeak friendly ping :)

**Comment by drazisil** [comment]
@pablofullana 

Can you write a test for this as well? It looks like we don't have one for Heroku, which is why this was never caught. I can write one as well, but it may be a while.

**Comment by drazisil** [comment]
Hmm. Can't see on mobile why the tests are failing, but will take a look tomorrow, thanks.

File: tests/env [hunk 1]
```diff
@@ -117,3 +117,6 @@ export CODEBUILD_RESOLVED_SOURCE_VERSION=""
 export CODEBUILD_SOURCE_VERSION=""
 export CODEBUILD_BUILD_ID=""
 export CODEBUILD_SOURCE_REPO_URL=""
+export HEROKU_TEST_RUN_BRANCH=""
+export HEROKU_TEST_RUN_ID=""
+export HEROKU_TEST_RUN_COMMIT_VERSION=""
```
[the remark this answers is not in the document]
Comment (pablofullana) [reply tests/env]: Not quite sure which is the one you are referring to. Those are the only 3 env vars  I had to add.

File: tests/test [hunk 2]
```diff
@@ -486,5 +486,14 @@ function test_cirrus () {
     assertURL "https://codecov.io/upload/v4?package=bash-tbd&token=&branch=master&commit=$TEST_DATA_GIT_COMMIT&build=239&build_url=&name=&tag=&slug=codecov%2Fci-repo&service=cirrus&flags=&pr=1&job=test"
 }
 
+function test_heroku () {
+    reset
+    export CI="true"
+    export HEROKU_TEST_RUN_BRANCH="master"
+    export HEROKU_TEST_RUN_ID="1"
+    export HEROKU_TEST_RUN_COMMIT_VERSION="$TEST_DATA_GIT_COMMIT"
+    assertURL "https://codecov.io/upload/v4?package=bash-tbd&token=&branch=master&commit=$TEST_DATA_GIT_COMMIT&build=1&build_url=&name=&tag=&slug=codecov%2Fcodecov-bash&service=heroku&flags=&pr=&job="
```
[the remark this answers is not in the document]
Comment (pablofullana) [reply tests/test]: CI seems to be working now. Which test repo are you referring to?

[the remark this answers is not in the document]
Comment (pablofullana) [reply tests/test]: CI seems to be working now. Which test repo are you referring to?

**Comment by relwell** [comment]
What do we need to do to get this to working? It doesn't even look like I can even pass `-N $HEROKU_TEST_RUN_COMMIT_VERSION` as an argument to get this working.

**Comment by drazisil** [comment]
@relwell I'll check on Monday, i think you are looking for `-C` though.

**Comment by dowtkdan** [comment]
Any update on this one? Happy to help where possible

**Comment by drazisil** [comment]
I've been on a leave of absence. 

@thomasrockhu can you take a look?

**Comment by thomasrockhu** [comment]
Can do, will take a look this week

**Comment by thomasrockhu** [comment]
@pablofullana please update the `SHA1SUM` file by running `shasum --algorithm 1 codecov > SHA1SUM` and I'd be happy to merge

**Comment by thomasrockhu** [comment]
Closing out in favor of https://github.com/codecov/codecov-bash/pull/348 in order to get this merged in.

**Pull request closed without merging**